bin/lib-gm

```diff
@@ -262,6 +262,12 @@ enforce_requirements() {
   if [ $((MAJOR)) -eq 0 ] && [ $((MINOR)) -lt 7 ]; then
     exit_with_error "stoml too old, install 0.7.0 or newer from https://github.com/freshautomations/stoml/releases"
   fi
+  if [ -z "$(which sort || echo)" ]; then
+    exit_with_error "missing sort, please install it"
+  fi
+  if ! test "$(echo "a\nb" | sort -Vr | sed -n '1p')" = "b"; then
```

Comment:
Ok, fair point, `sed` is in the list of requirements. But I really want to get rid of it one of these days.

Interestingly, we use `grep` extensively but it wasn't added to the requirements for some reason.

I guess, one of these days I'll open a "reduce requirements" issue.